HRTool/src/main/java/app/controllers/GoogleLoginVerifierController.java

```diff
@@ -1,143 +1,143 @@
-				    	
-				    	System.out.println("GLVC SCH username: "+((UserDetails)SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getUsername());
-				    	System.out.println("GLVC SCH authorities: "+((UserDetails)SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getAuthorities());
-				    	
-				    	System.out.println("GLVC AUTHORITIES: "+auth.getAuthorities());
-				    	
-				    	System.out.println("GLVC principal: "+SecurityContextHolder.getContext().getAuthentication().getPrincipal());
-				    	
-				    	return "redirect:/startPage.html";
-				    	
-				    }else{
-				    	throw new UserNotFoundException(payload.getEmail());
-				    }
-				    
-				   
-				  } else {
-				    System.out.println("Invalid ID token.");
-				  }
-				} else {
-				  System.out.println("Invalid ID token.");
-				}
-
-		} catch (GeneralSecurityException e) {
-			e.printStackTrace();
-		} catch (IOException e) {
-			e.printStackTrace();
-		}  	
-	
-		return "";
-	}
-	
-	
-	@RequestMapping(value = "/logout", method = RequestMethod.GET)
-	public String logOut(){
-		SecurityContextHolder.getContext().setAuthentication(null);
-		
-		return "redirect:/index.html";
-	}
-	
-
-	
-}
+package app.controllers;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import security.UserDetailsServiceImpl;
+import app.controllers.http.response.UserNotFoundException;
+import app.model.Employee;
+import app.repository.EmployeeRepository;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+@RestController
+@Configuration
+@ComponentScan("security")
+@RequestMapping("/signin")
+public class GoogleLoginVerifierController {
```

Comment:
This controller should NOT exist. Methods should be moved to employeeController.